Flytter v5-dokumentasjon til signering-docs (#181)
digipost/signature-api-client-dotnet: docs/_v5_x/1_setup.md

The library has several packages with the prefix `Digipost.Signature.Api.Client`. If you are using the portal cases, use `Digipost.Signature.Api.Client.Portal`, and for direct cases, use `Digipost.Signature.Api.Client.Direct`. 

> ## The documentation for sending and error handling is moved

Reviewer: ```suggestion
## The latest version of the documentation has moved 
```

>  The documentation for v5 is moved to [signering docs](https://signering-docs.readthedocs.io/en/latest/index.html). All older versions can still be browsed here.

Reviewer: ```suggestion
 The documentation for v5 has moved to [signering docs](https://signering-docs.readthedocs.io/en/latest/index.html). All older versions can still be browsed here.
```
